akka-distributed-data/src/main/scala/akka/cluster/ddata/protobuf/ReplicatedDataSerializer.scala

```diff
@@ -366,10 +413,28 @@ class ReplicatedDataSerializer(val system: ExtendedActorSystem)
     val longElements = new ArrayList[jl.Long]
     val otherElements = new ArrayList[dm.OtherMessage]
     var otherElementsMap = Map.empty[dm.OtherMessage, Any]
+    val actorRefElements = new ArrayList[ActorRef]
     orset.elementsMap.keysIterator.foreach {
       case s: String ⇒ stringElements.add(s)
       case i: Int    ⇒ intElements.add(i)
       case l: Long   ⇒ longElements.add(l)
+      case ref: ActorRef ⇒
+        actorRefFormat match {
+          case BothOldAndNewActorRefFormat ⇒
+            actorRefElements.add(ref)
+            val enclosedMsg = otherMessageToProto(ref)
+            otherElements.add(enclosedMsg)
+            // need the mapping back to the `other` when adding dots
+            otherElementsMap = otherElementsMap.updated(enclosedMsg, ref)
```

Comment:
In 2.5.14 it will include each ActorRef in both the new `actorRefElements` and the `otherElements`. In that way 2.5.14 can speak to both 2.5.13 and 2.5.15.